src/ContactListInfo.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Fragment, useState } from "react";
+
+function ContactListInfo({ email, sports }) {
+  const [isInfoVisible, setIsInfoVisible] = useState(false);
+  const handleInfoVisibility = () => {
+    setIsInfoVisible(!isInfoVisible);
+  };
+  return (
+    <>
+      <button className="info-button" onClick={handleInfoVisibility}>
+        {isInfoVisible ? "Hide Info" : "Show Info"}
+      </button>
+      {isInfoVisible ?<p className="email">{email}</p> : null}
+      {isInfoVisible
+        ? sports.map(({ id, name }) => {
+            return (
+              <Fragment key={id}>
+                <p className="sports-name">{name}</p>
+              </Fragment>
+            );
+          })
+        : null}
```

Comment:
Please explain the suggested changes in a voice.
```suggestion
      {isInfoVisible
        ?  (
        <>
         <p className="email">{email}</p>
         {sports.map(({ id, name }) => {
            return (
              <Fragment key={id}>
                <p className="sports-name">{name}</p>
              </Fragment>
            );
          })
       </>
       }) : null}
```